docs/root/configuration/listeners/network_filters/redis_proxy_filter.rst

```diff
@@ -70,3 +72,51 @@ The Redis proxy filter supports the following runtime settings:
 redis.drain_close_enabled
   % of connections that will be drain closed if the server is draining and would otherwise
   attempt a drain close. Defaults to 100.
+
+.. _config_network_filters_redis_proxy_fault_injection:
+
+Fault Injection
+---------------
+
+The Redis filter can perform fault injection. Currently, Delay and Error faults are supported.
+Delay faults delay a request, and Error faults respond with an error. Moreover, errors can be delayed.
+
+Note that the Redis filter does not check for correctness in your configuration- it is user's
+responsibility to make sure both the default and runtime percentages are correct! This is because
+percentages can be changed during runtime, and validating correctness at request time is expensive.
+If multiple faults are specified, the fault injection percentage should not exceed 100% for a given 
+fault and Redis command combination. For example, if you have two faults; one applying to GET at 60
+%, and one applying to all commands at 50%, that is a bad configuration as GET now has 110% chance of
```

Comment:
What happens in this case? It's not clear.

Reply:
Every request will get a fault. I'll explicitly state that.